src/service/resources-impl.js

```diff
@@ -1216,14 +1229,14 @@ export class Resources {
       request.newHeight = newHeight;
       request.newWidth = newWidth;
       request.force = force || request.force;
-      request.callback = opt_callback;
+      request.resolveCb = opt_resolveCb;
     } else {
       this.requestsChangeSize_.push(/** {!ChangeSizeRequestDef} */{
         resource,
         newHeight,
         newWidth,
         force,
-        callback: opt_callback,
+        resolveCb: opt_resolveCb,
```

Comment:
i don't see a need for renaming here? - was this a request from someone?
callback is a callback- - the purpose (resolving a promise) does not really matter